win32/mxe/copydlls.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
```

Comment:
Python 2 is EOL since 2019-12-31 and has been removed from many distributions. I couldn't see anything in this script that wouldn't work with python3 by a brief inspection, and in any case any such incompatibilities should be easily fixed.

Reply:
I know; this is just a temporary fix.

Reply:
@Tronic  Better now?